device/mellanox/x86_64-mlnx_msn3800-r0/Mellanox-SN3800-D100C12S2/hwsku.json

```diff
@@ -0,0 +1,346 @@
+            "default_brkout_mode": "2x50G[40G,25G,10G]"
+        },
+        "Ethernet88": {
+            "default_brkout_mode": "2x50G[40G,25G,10G]"
+        },
+        "Ethernet90": {
+            "default_brkout_mode": "2x50G[40G,25G,10G]"
+        },
+        "Ethernet92": {
+            "default_brkout_mode": "2x50G[40G,25G,10G]"
+        },
+        "Ethernet94": {
+            "default_brkout_mode": "2x50G[40G,25G,10G]"
+        },
+        "Ethernet96": {
+            "default_brkout_mode": "2x50G[40G,25G,10G]"
+        },
+        "Ethernet98": {
+            "default_brkout_mode": "2x50G[40G,25G,10G]"
+        },
+        "Ethernet100": {
+            "default_brkout_mode": "2x50G[40G,25G,10G]"
+        },
+        "Ethernet102": {
+            "default_brkout_mode": "2x50G[40G,25G,10G]"
+        },
+        "Ethernet104": {
+            "default_brkout_mode": "2x50G[40G,25G,10G]"
+        },
+        "Ethernet106": {
+            "default_brkout_mode": "2x50G[40G,25G,10G]"
+        },
+        "Ethernet108": {
+            "default_brkout_mode": "2x50G[40G,25G,10G]"
+        },
+        "Ethernet110": {
+            "default_brkout_mode": "2x50G[40G,25G,10G]"
+        },
+        "Ethernet112": {
+            "default_brkout_mode": "2x50G[40G,25G,10G]"
+        },
+        "Ethernet114": {
+            "default_brkout_mode": "2x50G[40G,25G,10G]"
+        },
+        "Ethernet116": {
+            "default_brkout_mode": "2x50G[40G,25G,10G]"
+        },
+        "Ethernet118": {
+            "default_brkout_mode": "2x50G[40G,25G,10G]"
+        },
+        "Ethernet120": {
+            "default_brkout_mode": "2x50G[40G,25G,10G]"
+        },
+        "Ethernet122": {
+            "default_brkout_mode": "2x50G[40G,25G,10G]"
+        },
+        "Ethernet124": {
+            "default_brkout_mode": "2x50G[40G,25G,10G]"
+        },
+        "Ethernet126": {
+            "default_brkout_mode": "2x50G[40G,25G,10G]"
+        },
+        "Ethernet128": {
+            "default_brkout_mode": "2x50G[40G,25G,10G]"
+        },
+        "Ethernet130": {
+            "default_brkout_mode": "2x50G[40G,25G,10G]"
+        },
+        "Ethernet132": {
+            "default_brkout_mode": "2x50G[40G,25G,10G]"
+        },
+        "Ethernet134": {
+            "default_brkout_mode": "2x50G[40G,25G,10G]"
+        },
+        "Ethernet136": {
+            "default_brkout_mode": "2x50G[40G,25G,10G]"
+        },
+        "Ethernet138": {
+            "default_brkout_mode": "2x50G[40G,25G,10G]"
+        },
+        "Ethernet140": {
+            "default_brkout_mode": "2x50G[40G,25G,10G]"
+        },
+        "Ethernet142": {
+            "default_brkout_mode": "2x50G[40G,25G,10G]"
+        },
+        "Ethernet144": {
+            "default_brkout_mode": "2x50G[40G,25G,10G]"
+        },
+        "Ethernet146": {
+            "default_brkout_mode": "2x50G[40G,25G,10G]"
+        },
+        "Ethernet148": {
+            "default_brkout_mode": "1x10G[100G,50G,40G,25G]"
```

Comment:
Due to https://github.com/Azure/sonic-buildimage/issues/7958 these modes will need to be added to the list of supported breakout modes of the `platform.json` file for the 3800.

Reply:
I believe the 1x10G is the only missing one but please double check.

Reply:
@allexrallen platform.json is added with 1x10G support. You can see it in the changed files.

Reply:
Looks good to me.